### ISSUE TITLE
Duplicate of #2

### ISSUE THREAD

**Issue by sho7a** [jbaubree/vite-ssg-sitemap#124]
Version: 0.3.1

```xml
<urlset>
  <url>
    <loc>http://localhost/index</loc>
    <lastmod>2022-08-07T15:35:47.307Z</lastmod>
    <changefreq>daily</changefreq>
    <priority>1.0</priority>
  </url>
</urlset>
```

**Comment by jbaubree** [comment]
Thanks for reporting, this is fixed in v0.3.2

**Comment by sho7a** [comment]
It's working, thanks!
